.ado/windows-vs-pr.yml

```diff
@@ -535,11 +539,8 @@ jobs:
   - template: templates/e2e-test-job.yml # Template reference
     parameters:
       name: E2ETest
-      pool:
```

Comment:
Please revert this.
Even though today E2E can only run on windows-2019, this may change, and that's exactly what it's a parameter.

If you want to prevent this value to be changed, please add a warning comment above this line.

Reply:
add comments on both files